src/processing/handlers/BoardStateHandler.java

```diff
@@ -15,15 +16,23 @@
  */
 
 public class BoardStateHandler implements INotificationHandler {
-	public void onMessageReceived(String message) {
-		
+	
+	public static void handleBoardState(String message) {
 		try {
-				BoardState boardState = (BoardState)Serialize.deSerialize(message);
-				ClientBoardState.maps = boardState;
+			BoardState boardState = (BoardState)Serialize.deSerialize(message);
+			ClientBoardState.maps = boardState;
 		} catch (ClassNotFoundException e) {
 			//Log the exception
```

Comment:
You wrote but did not log it !!

Reply:
This is satchit's file, see the author, I just made some changes